cogctl/cli/profile.py

```diff
@@ -42,8 +42,9 @@ def create(state, name, host, port, secure, rest_user, rest_password):
     """
```

Comment:
All parameters (whether Options or Arguments) can have a `required` keyword argument. In any event, I've turned them all into arguments now.